### PR TITLE
Fix vitest config after prettier format to allow applying front modules

### PR DESCRIPTION
**Pull request by pascalgrimaud**
- related to https://github.com/jhipster/jhipster-lite/issues/11106

### CODE REVIEW

File: src/main/resources/generator/typescript/vitest.config.ts.mustache [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default defineConfig({
       },
       include: ['src/main/webapp/**/*.ts?(x)'],
       exclude: [
+        // keep this comment to be able to apply other front module after prettier format
```
Comment (murdos) [inline src/main/resources/generator/typescript/vitest.config.ts.mustache:32]: I'm not really found on this comment that will stay on generated project, just for generation purpose.
I'll try another solution and let you know.

Comment (pascalgrimaud) [reply src/main/resources/generator/typescript/vitest.config.ts.mustache:32]: Agree, it was just the quickest fix in the mean time. Feel free to propose better solution, and rework the replace after lines for each front

Comment (murdos) [reply src/main/resources/generator/typescript/vitest.config.ts.mustache:32]: Here's an alternative (that is not perfect either): https://github.com/jhipster/jhipster-lite/pull/11189

Comment (pascalgrimaud) [reply src/main/resources/generator/typescript/vitest.config.ts.mustache:32]: yes, but it's better

**Pull request merged**